### PR TITLE
[No QA] Fix api usage inside `AddEncryptedAuthToken` migration

### PR DESCRIPTION
**Pull request by kidroca**
@marcaaron

### Details

Fixes a problem related to migrating usages to `import *`
Looks like it was only applied partially here 

### Fixed Issues
$ N/A

### Tested On

- [x] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

#### Web
image

image

### CODE REVIEW

**Review by ctkochan22 — DISMISSED** [review]
Simple change. @marcaaron let you merge

File: src/libs/migrations/AddEncryptedAuthToken.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import _ from 'underscore';
 import Onyx from 'react-native-onyx';
 import Log from '../Log';
 import ONYXKEYS from '../../ONYXKEYS';
-import * as reauthenticate from '../API';
+import * as Api from '../API';
```
Comment (marcaaron) [inline src/libs/migrations/AddEncryptedAuthToken.js:5]: Mind changing this to `API` for consistency?

**Review by marcaaron — CHANGES_REQUESTED** [review]
Thanks! This was on my list of things to fix, but never found the time to do it. Guessing not many were affected, but it was a careless mistake.

**Review by ctkochan22 — APPROVED** [review]

**Review by marcaaron — APPROVED** [review]

**Comment by marcaaron** [comment]
Looks great, thanks again!

**Pull request merged**